src/libraries/System.Runtime/ref/System.Runtime.cs

```diff
@@ -2194,6 +2194,53 @@ public partial struct Guid : System.IComparable, System.IComparable<System.Guid>
         public static bool TryParseExact([System.Diagnostics.CodeAnalysis.NotNullWhenAttribute(true)] string? input, [System.Diagnostics.CodeAnalysis.NotNullWhenAttribute(true)] string? format, out System.Guid result) { throw null; }
         public bool TryWriteBytes(System.Span<byte> destination) { throw null; }
     }
+    public readonly partial struct Half : System.IComparable, System.IComparable<System.Half>, System.IEquatable<System.Half>, System.IFormattable
+    {
+        private readonly int _dummyPrimitive;
+        public static System.Half Epsilon { get { throw null; } }
+        public static System.Half MaxValue { get { throw null; } }
+        public static System.Half MinValue { get { throw null; } }
+        public static System.Half NaN { get { throw null; } }
+        public static System.Half NegativeInfinity { get { throw null; } }
+        public static System.Half PositiveInfinity { get { throw null; } }
```

Comment:
Shouldn't these values be constants?

Reply:
Discussed in https://github.com/dotnet/runtime/pull/37630#discussion_r438465609

Reply:
They can't be actual constants either because `constant` can only be a limited set of types in IL and in C#.